bin.js

```diff
@@ -22,7 +22,12 @@ var i = argv.indexOf('--')
 var conf = argv.slice(i+1)
 argv = ~i ? argv.slice(0, i) : argv
 
-var config = require('ssb-config/inject')(process.env.ssb_appname, minimist(conf))
+var config = require('ssb-config/inject')(
+  process.env.ssb_appname,
+  Object.assign(
+    minimist(conf),
+    { host: process.env.ssb_host }
```

Comment:
don't put a hack like this here... this should already work in ssb-config. are you sure it's needed?

Reply:
It may no longer be needed, at the time I think this was a side-effect of how ssb_host was being read / overwritten.